### ISSUE TITLE
Improve performance of this step in Bitrise

### ISSUE THREAD

**Issue by nickskull** [nodes-ios/bitrise-step-nodes-custom-script#23]
Currently, everytime bitrise interacts with this step it has to load all dependencies, do `bundle install` and other which take considerable amount of time (~20-30 seconds each time it is run), which we could optimize away.

We could either make this step do all the logic we need and have it as the final step in the Bitrise workflow or somehow check/make sure the dependencies are not being installed everytime.

**Comment by chriscombs** [comment]
I think the only reason we need the bundle install is for the badge tool. But the reason it installs every time is because every new bitrise fastlane step tries to reconfigure everything.

**Comment by nickskull** [comment]
@chriscombs Yeah, but I can see us in the future using more fastlane plugins, which is fine imho. It's just the last thing you said, where it reconfigures everything that I'd like to figure out how to optimize.
